core/src/cluster_info.rs

```diff
@@ -1834,7 +1833,7 @@ impl ClusterInfo {
 
                     let stakes: HashMap<_, _> = match bank_forks {
                         Some(ref bank_forks) => {
-                            staking_utils::staked_nodes(&bank_forks.read().unwrap().working_bank())
+                            bank_forks.read().unwrap().working_bank().staked_nodes()
```

Comment:
This is only somewhat related, but I think these should be looking at the rooted bank.

Reply:
can we make that change in a separate commit?
just in case it has unknown consequences

Reply:
Yes, thats' fine.